pkg/cmd/pulumi/new.go

```diff
@@ -183,26 +191,28 @@ func runNew(ctx context.Context, args newArgs) error {
 
 		// Set the org name for future use.
 		orgName = parts[0]
-
 		projectName := parts[1]
-		if err := validateProjectName(ctx, b, orgName, projectName, args.generateOnly, opts); err != nil {
-			return err
-		}
```

Comment:
"Validate project name errors if the project already exists."

Removing this check as we are checking the --stack argument so we don't care if the project already exists.